core/nodepool/config/templates/stack-template.json

```diff
@@ -398,6 +418,29 @@
                   "Resource": "*"
                 },
                 {{end}}
+                {{if .NodeDrainer.Enabled }}
+                {
+                  "Action": [
+                    "sns:Publish"
+                  ],
+                  "Effect": "Allow",
+                  "Resource": { "Ref": "{{.LogicalName}}NodeDrainerLHT" }
+                },
+                {
+                  "Action": [
+                    "autoscaling:CompleteLifecycleAction",
+                    "autoscaling:DescribeAutoScalingInstances",
+                    "autoscaling:DescribeLifecycleHooks"
```

Comment:
```
     "autoscaling:DescribeAutoScalingInstances" 
     "autoscaling:DescribeLifecycleHooks"
```

do not support condition keys: http://docs.aws.amazon.com/autoscaling/latest/userguide/control-access-using-iam.html#resource-level-permissions-auto-scaling